docs/docs_src/kafka/security/sasl_gssapi.py

```diff
@@ -0,0 +1,9 @@
+import ssl
```

Comment:
Why you added this file, but didn't use it anywhere?

Reply:
I included this file in the documentation, but I already wrote the test earlier, [here it is](https://github.com/airtai/faststream/pull/1633/files#diff-b047124b26c2b25687a87a0062acb16d8822abd5f8addb0144db3de8c2864631R24). 
Is this enough?

Reply:
Can u a test for it too? https://github.com/airtai/faststream/blob/main/tests/docs/kafka/test_security.py

Reply:
done